configs/resident/settings_mp.yaml

```diff
@@ -74,7 +74,10 @@ models:
   - write_data_dictionary
   - track_skim_usage
   - write_trip_matrices
+  # - write_to_datalake
+  - update_tables
```

Comment:
This pull request doesn't include the [changes to extensions\update_tables.py](https://github.com/SANDAG/ABM/pull/279/files#diff-76718fc227d6890023760ef2d4fc2d23ed5ba384a171a9a059d4840e3ed36873) to get that step to work with ActivitySim 1.3. Without it the model will fail at that step.

Reply:
Thanks for pointing it out! I reverted these changes. We can continue keeping update_tables.py out of this repo.